blog/2024-02-27-extraction-matters-most/index.mdx

```diff
@@ -0,0 +1,135 @@
+---
+slug: extraction-matters-most
+title: Extraction Matters Most
+image: ./heatmap_extractor.png
+description: |
+  Experimental measurements of a variety of options in a RAG pipeline lead
+  to the conclusion that how you extract content from documents matters most.
+keywords:
+  - retrieval augmented generation
+  - RAG experimental evaluation
+authors: [ben]
+---
+
+There is a lot of content on getting started with Retrieval Augmented Generation (RAG), and numerous posts on various forms of “advanced” RAG.
+LangChain lists 8 different PDF loaders, 4 different text splitters, and a variety of techniques you can add such as Parent-Document Retrieval and Multi-Vector indexing.
+It is easy to be paralyzed with doubt – which of these should you use?
+
+![RAG configuration choices](./configurations.svg)
+
+If you’re wondering which of these techniques are best for your domain, the long answer is – you should understand which techniques may help your documents and run experiments on your data to see which of those help.
```

Comment:
Is it worth adding a "short answer" / TLDR in the intro the explains the title? Something explaining for short-attention-spans that we found that extraction (and to a lesser extent indexing) are what matter?